### DISCUSSION TITLE
Update for Legion 7.0?

### DISCUSSION THREAD

**Discussion by PaniniGelato** [SFX-WoW/Masque#4] [Questions] [answered]
Hi,
I am using Masque for years.  It is a greate addon and thank you for your work.
Do u have any plan about update for Legion 7.0?

**Comment by StormFX** [comment, marked as answer]
It works fine if you enabled out-of-date add-ons. I'm currently dealing with the passing of my mother so it'll be a week or so.

**Comment by PaniniGelato** [comment]
I am sorry and hope you are doing well.  
RIP

**Comment by StormFX** [comment]
I'm OK. Thanks for the thought. :)
